crypto/pkcs12/p12_attr.c

```diff
@@ -125,5 +125,5 @@ void PKCS12_SAFEBAG_set0_attrs(PKCS12_SAFEBAG *bag, const STACK_OF(X509_ATTRIBUT
     if (bag->attrib != attrs)
        sk_X509_ATTRIBUTE_free(bag->attrib);
 
-    bag->attrib = (STACK_OF(X509_ATTRIBUTE*))attrs;
+    bag->attrib = (STACK_OF(X509_ATTRIBUTE) *)attrs;
```

Comment:
Hmmm. - this function doesn't quite look right. The cast is only necessary here because we are casting away the "const". But it probably should not be const at all, because we eventually plan to "free" this value. So I think the correct fix here is to remove the cast altogether and adjust the function signature to remove "const". This is ok because this function is newly added to master and doesn't exist in other branches - so backwards compatibility is not a concern.